### ISSUE TITLE
In our XML, <url>...</url> tags are used for urls in the text.  They are not hyperlinked by the client.  

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/PEP-Web-User-Interface#727]
Note the URL tag does not have an attribute with a target because the text itself is a target.

This came to light when I was reviewing the latest IJP on PEP-Web.  It is in fact in @davidtuckett  paper there are a lot of links in the biblio (bib) which aren't linked.  Of course, in these cases the reference linker linked them anyway, using other data from the reference.  But these can occur in other locations and should always be hyperlinked.

There are over 25,000 of these <url> tags in the PEP archive database.

Here's an short and easy example, though a bit trivial"
[https://stage.pep-web.org/search/document/PPSY.040.0010A?q=PPSY.040.0010A](https://stage.pep-web.org/search/document/PPSY.040.0010A?q=PPSY.040.0010A)

The xml:

```
<body>
<p>Would you like to know when the next issue of your favorite APA journal will be available online? This service is now available to you. Sign up at <url>https://my.apa.org/portal/alerts/</url> and you will be notified by e-mail when issues of interest to you become available!</p>
<pb><n>10</n></pb>
</body>
```

**Comment by ocappello** [comment]
can we activate them so that they are hyperlinked? it doesn't look great if they are not.

**Comment by nrshapiro** [comment]
> can we activate them so that they are hyperlinked? it doesn't look great if they are not.

Of course...just need Jordan to do that in the client.  just approve and I'll work with him on it.

**Comment by nrshapiro** [comment]
@jordanallen-dev @ocappello @davidtuckett 

Here's a good and very important example of this I've just run into in the latest issue of IJP.  The editor's article about IJPOpen attempts to post two links at the end:

```
<h1 id="H0001" align="center">IJP Open: How it Works</h1>
<p><url>https://pep-web.org/browse/</url></p>
<p>IJPOPEN/volumes <url>http://www.theijp.org/ijp-open/</url></p>
```
of course, per this issue above, the links don't work.  And given the prefix text of the second paragraph, it almost looks like one "wrapped" link on PEP-Web (if you don't look carefully, which I didn't at first).

<img width="1121" alt="image" src="https://user-images.githubusercontent.com/4837554/222259647-fb0031d4-2a4d-4c15-b8b3-991f26bbceea.png">

**Comment by jordanallen-dev** [comment]
> @jordanallen-dev @ocappello @davidtuckett
> 
> Here's a good and very important example of this I've just run into in the latest issue of IJP.
> 
> https://pep-web.org/browse/document/IJP.104.0004A?page=P0004
> 
> The editor's article about IJPOpen attempts to post two links at the end:
> 
> ```
> <h1 id="H0001" align="center">IJP Open: How it Works</h1>
> <p><url>https://pep-web.org/browse/</url></p>
> <p>IJPOPEN/volumes <url>http://www.theijp.org/ijp-open/</url></p>
> ```
> 
> of course, per this issue above, the links don't work. And given the prefix text of the second paragraph, it almost looks like one "wrapped" link on PEP-Web (if you don't look carefully, which I didn't at first).
> 
> <img alt="image" width="1121" src="https://user-images.githubusercontent.com/4837554/222259647-fb0031d4-2a4d-4c15-b8b3-991f26bbceea.png">

Implementation in progress

https://www.notion.so/Transform-url-into-a-during-XML-HTML-bc6b41369fb04e669fbf02b866672c9f?pvs=4

**Comment by jordanallen-dev** [comment]
@davidtuckett @ocappello @nrshapiro 

Implementation complete and live on stage

https://stage.pep-web.org/browse/document/IJP.104.0004A?page=P0004

Notice Francis' email doesn't link correctly

That's because in the XML their email configured as a <url> as well but the XML -> HTML converter has no way of knowing that it needs to add the special `mailto:` prefix

**Comment by nrshapiro** [comment]
@davidtuckett @ocappello @jordanallen-dev @SophieMBennett 

Another example in the link issue David and I were talking about in [IJP.101.1106A](https://stage.pep-web.org/search/document/IJP.101.1106A?page=P1106&q=IJP.101.1106A).

It's marked as webx in the xml, with a URL link, but it's not styled or linked by the client

and no spacing is added between the two webx links by the style sheet)

Per Jordan:
 -> webx parsing is misconfigured in the XML -> HTML

(He's looking into fixing this.)

Note that the links are in the Caption, not the figure, and they are set correctly.  Once the webx parsing is fixed we should look at the styling.

It would be nice to have links to external videos work like our other videos, but that should be handled as a new feature and would have to be done by Jordan (in the client, and potentially reconfigured by Aptara in the JATS->XML conversion.)

**Comment by nrshapiro** [comment]
Note @jordanallen-dev 
1) we still need a space between the links in the caption
2) The feature request of viewing the video embedded should be raised as a new issue

<img width="781" alt="image" src="https://user-images.githubusercontent.com/4837554/234300093-7dd020cf-9868-4fe3-88c4-39cf9134b245.png">
